presto-native-execution/presto_cpp/main/connectors/PrestoToVeloxConnector.cpp

```diff
@@ -1558,11 +1561,36 @@ IcebergPrestoToVeloxConnector::toVeloxInsertTableHandle(
           icebergOutputTableHandle->partitionSpec, typeParser),
       pool,
       toVeloxFileFormat(icebergOutputTableHandle->fileFormat),
-      nullptr,
+      std::move(sortedBy),
       std::optional(
           toFileCompressionKind(icebergOutputTableHandle->compressionCodec)));
 }
 
+std::vector<velox::connector::hive::iceberg::IcebergSortingColumn>
+IcebergPrestoToVeloxConnector::toIcebergSortingColumns(
+    protocol::List<protocol::iceberg::SortField> sortFields,
+    const protocol::iceberg::PrestoIcebergSchema& schema) const {
+  std::vector<connector::hive::iceberg::IcebergSortingColumn> sortedBy;
+  sortedBy.reserve(sortFields.size());
+  for (const auto& sortField : sortFields) {
+    velox::core::SortOrder veloxSortOrder(
```

Comment:
Please can you leave a comment stating this matches asc/desc, nulls first/nulls last.